nifi-ngsi-bundle/nifi-ngsi-processors/src/test/java/org/apache/nifi/processors/ngsi/TestNGSIToPostgreSQL.java

```diff
@@ -765,7 +765,7 @@ public void testValuesForInsertColumnForNgsiLd() throws IOException {
 
         long creationTime = 1562561734983l;
 
-        String expectedValuesForInsert = "('urn:ngsi-ld:NifiTest:Test01','NifiTest',$$urn:ngsi-ld:RelationTest:Test03$$,$$ES$$,'2020-09-29T09:00:00Z',null,$$urn:ngsi-ld:RelationTest:Test02$$,$$USA$$,'2020-09-29T09:00:00Z',null,$$[3.63969,43.43358]$$,'2020-09-29T09:00:00Z',null,$$test 01$$,'2020-09-29T09:00:00Z',null,'2019-07-08T04:55:34.983Z',null,'2020-09-29T09:00:00Z',null)";
+        String expectedValuesForInsert = "('urn:ngsi-ld:NifiTest:Test01','NifiTest',$$urn:ngsi-ld:RelationTest:Test03$$,$$ES$$,'2020-09-29T09:00:00Z',null,$$urn:ngsi-ld:RelationTest:Test02$$,$$USA$$,'2020-09-29T09:00:00Z',null,$$Point$$,'2020-09-29T09:00:00Z',43.43358,3.63969,null,$$test 01$$,'2020-09-29T09:00:00Z',null,'2019-07-08T04:55:34.983Z',null,'2020-09-29T09:00:00Z',null)";
```

Comment:
is it up to date? we should have more columns?

Reply:
No. I update it.